pdp/context.go

```diff
@@ -105,7 +105,7 @@ func NewContext(c *LocalContentStorage, count int, f func(i int) (string, Attrib
 // Requirements for the storage are the same as NewContext function has.
 // The request umarshaled to sequence of attributes as descirbed by
 // (Un)MarshalRequest* functions help.
-func NewContextFromBytes(c *LocalContentStorage, b []byte) (*Context, error) {
+func NewContextFromBytes(c *LocalContentStorage, b []byte, currentContext *Context) (*Context, error) {
```

Comment:
This changes the function signature which could break other clients of this code. Can you make a new method instead?